### ISSUE TITLE
Pre-populate calendar without having to use modifiers

### ISSUE THREAD

**Issue by cmrabet** [gpbl/react-day-picker#247]
Is there anyway to "set" the days that I want selected in one shot that is not via modifiers? I'm asking this because using modifiers is slow when managing year calendars.

I have an X amount of dates stored in my database that can have different types (holiday, non workable day, etc..). I want to populate a year calendar with them using different colors depending on their type. So far I've achieved this via modifiers like so in my `render` method:

```
	// Day modifiers
		const modifiers={
	        sunday:(cellDate)=>{
	          return cellDate.getDay()==0;
	        },
	        holiday: (cellDate)=>{
	          const date=moment(cellDate).format('YYYY-MM-DD');
	          const day=_.find(this.props.calendar,(e)=>{return moment(e.date).format('YYYY-MM-DD')==date;})
	          if(day) return day.type==1;
	          else return false;
	        },
	        nonWorkable: (cellDate)=>{
	          const date=moment(cellDate).format('YYYY-MM-DD');
	          const day=_.find(this.props.calendar,(e)=>{return moment(e.date).format('YYYY-MM-DD')==date;})
	          if(day) return day.type==2;
	          else return false;
	        }         
	    };  
```

Where `this.props.calendar` is a prop populated by the parent component with the days fetched from the database via a reducer (using redux). 

This is very inefficient since I have to run 365 times for X number of days for each modifier each time the component renders. It would be very nice to have a method like `setDays()` that admitted an array of days and their class values so we could sed in one shot only once.

Can I do this now?

Thanks.

**Comment by gpbl** [comment]
Sadly now this is not possible. I'm aware of this design performance issues, in particular this case would be solved with https://github.com/gpbl/react-day-picker/issues/181#issuecomment-225684329. I'd like to know your feedback on that issue!
